.github/workflows/ci.yml

```diff
@@ -95,3 +96,82 @@ jobs:
 
       - name: Run specs
         run: npm run sass-spec -- --impl libsass -c ../sassc/bin/sassc
+
+  dart_sass_js:
+    name: "JS API | Dart Sass | ${{ matrix.os }}"
```

Comment:
Does this sass-spec repo really need to test dart-sass on all OS ? The dart-sass repo already does it.

Reply:
When we add new tests here we need to verify that they pass on all OSes. It would be pretty bad if we added a test that failed on Windows, but only saw that failure the next time a pull request was sent out against Dart Sass.